.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+
+jobs:
+  test:
+    uses: ./.github/workflows/test.yml
+
+  integration-tests:
+    uses: ./.github/workflows/integration_tests.yml
+
+  build-check:
+    uses: ./.github/workflows/build.yml
+    with:
+      push_artifacts: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)